src/mlpack/methods/dbscan/ordered_point_selection.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file ordered_point_selection.hpp
+ * @author Kim Sang-Yeon
+ *
+ * sequentially select the next point for DBSCAN.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_DBSCAN_ORDERED_POINT_SELECTION_HPP
+#define MLPACK_METHODS_DBSCAN_ORDERED_POINT_SELECTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace dbscan {
+
+/**
+ * This class can be used to sequentially select the next point to use for DBSCAN.
+ */
+class OrderedPointSelection
+{
+ public:
+  /**
+   * Select the next point to use, sequentially.
+   *
+   * @param unvisited Bitset indicating which points are unvisited.
+   * @param data Unused data.
+   */
+  template<typename MatType>
+  size_t Select(const size_t point,
```

Comment:
Should we use a static method here, to make this more readable/accessible  ``PointSelectionPolicy::Select(...)``?

Reply:
Okay, I will do.